drivers/sensor/lsm6ds0/lsm6ds0.h

```diff
@@ -314,7 +314,7 @@
 
 #define SENSOR_G_DOUBLE		(SENSOR_G / 1000000.0)
 #define SENSOR_PI_DOUBLE	(SENSOR_PI / 1000000.0)
-#define SENSOR_DEG2RAD_DOUBLE	(180.0 / SENSOR_PI_DOUBLE)
+#define SENSOR_DEG2RAD_DOUBLE	(SENSOR_PI_DOUBLE / 180)
```

Comment:
Is there a reason you're not using `sensor_degrees_to_rad` defined in `include/drivers/sensor.h`?